public/js/index.js

```diff
@@ -0,0 +1,12 @@
+(function () {
+  window.addEventListener("load", function() {
+    var storySummary = document.getElementById('storySummary');
+    var headlines = document.getElementById('headlines');
+    var controller = new NewsManagerController(storySummary,
+                              headlines,
+                              new NewsManagerModel(),
+                              new NewsManagerView(),
+                              new ApiManager());
+    controller.api.apiRequest(controller, controller.createStories);
```

Comment:
Can this invocation be moved so it's inside the controller? This let's the controller control how the app is started.